authservice/auth_service.go

```diff
@@ -304,9 +304,11 @@ func (r *RemoteServiceRegLoaderImpl) LoadServices() ([]ServiceReg, error) {
 	}
 
 	validate := validator.New()
-	err = validate.Struct(services)
-	if err != nil {
-		return nil, fmt.Errorf("error validating service data: %v", err)
+	for _, service := range services {
```

Comment:
You can probably combine this for loop with the one below to avoid iterating twice